src/ApiPort/Resources/LocalizedStrings.resx

```diff
@@ -231,4 +232,13 @@ To learn more about how this tool works, including the data we are collecting, g
   <data name="CmdHelpSuppressBreakingChange" xml:space="preserve">
     <value>Specifies a breaking change (by ID) to suppress during breaking change analysis. Any breaking changes with IDs specified for suppression will not be reported.</value>
   </data>
+  <data name="CmdAnalyzeHelp" xml:space="preserve">
+    <value>Analyzes specified files/directories for IL and determines the APIs that are used. This information is analyzed for portability concerns or known breaking changes (depending on input parameters) and returns a report that gives a summary of issues and possible steps forward.</value>
```

Comment:
This line is long; does it wrap funny? Should we maybe split it up at <= 80 characters per line?